packages/app/src/assets/index.css

```
@@ -0,0 +1,101 @@
+}
+
+[data-theme='dark'] {
+  --theme-color: white;
+  --theme-background-0: #1f1f1f;
+  --theme-background-1: #212121;
+  --theme-background-2: #252525;
+  --theme-selected: #2f2f2f;
+  --theme-hover: #2f2f2f80;
+  --theme-border: #424242;
+  --theme-primary: #3b82f6;
+  --theme-primary: #ea9f07;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--theme-primary is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)